docs/pages/components/Button.mdx

```diff
@@ -78,6 +78,30 @@ When you need a semantic button with the visual appearance of a link.
 <Button variant="tag">Tag</Button>
 ```
 
+### Badge
+
+```jsx example
+<Button variant="badge">Badge</Button>
```

Comment:
I'm not seeing a hover state when I hover over a badge.

https://github.com/user-attachments/assets/5d04aa75-64bc-4088-bd5f-129296aa5750

Reply:
Interesting, thanks for pointing that out. I'll look into this!